scripts/plot_dispatch.py

```diff
@@ -0,0 +1,119 @@
+import os
+import sys
+
+import matplotlib.dates as mdates
+import matplotlib.pyplot as plt
+import oemoflex.tools.plots as plots
+from oemof_flexmex.model_config import plot_labels, colors_odict
+import pandas as pd
+from addict import Dict
+
+def sum_demands(data, bus_name, demand_name):
+    d_demand = pd.DataFrame()
+    for i in data.columns:
+        if demand_name in i[1]:
+            d_demand[i] = data[i]
+            data.drop(columns=[i], inplace=True)
+    total_demand = d_demand.sum(axis=1)
+    data[(bus_name, bus_name + '-demand', 'flow')] = total_demand
+    return data
+
+if __name__ == "__main__":
+
+    paths = Dict()
+    paths.postprocessed = sys.argv[1]
+    paths.plotted = sys.argv[2]
+
+    # create the directory plotted where all plots are saved
+    if not os.path.exists(paths.plotted):
+        os.makedirs(paths.plotted)
+
+    timeseries_directory = os.path.join(paths.postprocessed, "oemoflex-timeseries")
+    timeseries_files = os.listdir(timeseries_directory)
+
+    # select timeframe
+    timeframe = [
+        ("2019-01-01 00:00:00", "2019-01-31 23:00:00"),
+        ("2019-07-01 00:00:00", "2019-07-31 23:00:00"),
+    ]
+
+    carriers = ["electricity.csv", "heat_decentral", "heat_central"]
+
+    for carrier in carriers:
+
+      selected_timeseries_files = [file for file in timeseries_files if carrier in file]
```

Comment:
```suggestion
      selected_timeseries_files = [file for file in timeseries_files for carrier in carriers if carrier in file]
```

Reply:
You can put the "carrier" loop in the same list comprehension.

Reply:
There seems to be an inconsistent indentation. Some indentations are only two spaces instead of four. Do we have linting with black set up already?